### PR TITLE
Typo fixes(ExpandableVariables.md)

### PR DESCRIPTION
**Pull request by Luro223**

### CODE REVIEW

**Review by isaak654 — COMMENTED** [review]
Please review a3ec0fbf57325da0ba8f16415d3bbd1bf1d5a5c1 (thumbs up for approval).

**Comment by isaak654** [comment]
@Luro223 :point_up_2:

File: Content/ExpandableVariables.md [hunk 1]
```diff
@@ -74,6 +74,6 @@ For example:
     Reg.Cookies=%USERPROFILE%\Cookies
 ```
 
-Note that "Ovr."-style overrides (described above) will cause Sandboxie to not look in the registry at all. On the other hand, Sandboxie only considers "Reg."-style fallbacks if the expanded variable cannot be found in the registry. This means that if both Ovr.X and Reg.X are specified for the same variable X, the Ovr.X form will always apply when X is expanded, and the Reg.X form will never apply.
+Note that "Ovr." style overrides (described above) will skip the registry addressing for Sandboxie. On the other hand, Sandboxie only checks "Reg." style fallbacks if the expanded variable cannot be found in the registry. This means that if both Ovr.X and Reg.X are specified for the same variable X, the Ovr.X form will always apply when X is expanded, and the Reg.X form will never apply.
```
Comment (isaak654) [inline Content/ExpandableVariables.md]: The term "addressing" seems a bit too technical, is there a simpler one?

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Content/ExpandableVariables.md (hunk 1 above)
Comment (Luro223) [reply Content/ExpandableVariables.md]: will force/make/cause Sandboxie (to) ignore the registry (lookup)

Comment (isaak654) [inline Content/ExpandableVariables.md]:
```suggestion
Note that "Ovr." style overrides (described above) will cause Sandboxie to ignore the registry. On the other hand, Sandboxie only checks "Reg." style fallbacks if the expanded variable cannot be found in the registry. This means that if both Ovr.X and Reg.X are specified for the same variable X, the Ovr.X form will always apply when X is expanded, and the Reg.X form will never apply.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Content/ExpandableVariables.md (hunk 1 above)
Comment (isaak654) [reply Content/ExpandableVariables.md]: https://github.com/sandboxie-plus/sandboxie-docs/pull/79#pullrequestreview-1121933400

File: Content/ExpandableVariables.md [hunk 2]
```diff
@@ -30,15 +30,15 @@ The following table lists the variables that Sandboxie recognizes.
...
+Global templates are part of the Sandboxie installation and located in the file _Templates.ini_ in the Sandboxie installation folder. Additional local templates may be added to [Sandboxie Ini](SandboxieIni.md). Any template may reference template variables in the form _%Tmpl.SomeVariableName%_. These variable names are not built into the core of Sandboxie. They must be defined in _Templates.ini_ or _Sandboxie.ini_ in a [TemplateSettings] section.
 
 ### Overriding Variables
 
-Any of the variables in the table above, including any of the [Shell Folders](ShellFolders.md) variable and any template variables, can be overridden by the [Sandboxie Ini](SandboxieIni.md) configuration file. To override a variable, add a setting with an **Ovr.** prefix.
+Any of the variables in the table above, including any of the [Shell Folders](ShellFolders.md) variables and any of the template variables, can be overridden by the [Sandboxie Ini](SandboxieIni.md) configuration file. To override a variable, add a parameter prefixed with **Ovr.**.
```
Comment (isaak654) [inline Content/ExpandableVariables.md]:
```suggestion
Any of the variables in the table above, including the [Shell Folders](ShellFolders.md) and template variables, can be overridden by the [Sandboxie Ini](SandboxieIni.md) configuration file. To override a variable, add a parameter prefixed with **Ovr.**.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — APPROVED** [review]

**Pull request merged**